### ISSUE TITLE
Add Quick Update to elements

### ISSUE THREAD

**Issue by adamwintle** [BobRay/Orphans#2]
Is it possible to add the right-click menu item for Quick Update, like you get in the main Element tree?

**Comment by BobRay** [comment]
I'm afraid I don't understand this. You want to Quick Update an orphan object?

**Comment by adamwintle** [comment]
Yes, for example, this Chunk called `homepage-tag` is apparently orphaned, but I'm not 100% sure of the code in the chunk; so I'd have to go into the Elements tab and drill down to find that chunk, then right-click in the tree to Quick Update Chunk to view the content without leaving the Orphans view I'm on.

![screenshot 2015-09-03 12 45 30](https://cloud.githubusercontent.com/assets/167075/9651366/f17ebfb2-5239-11e5-8c38-e5a60c7e2576.png)

To speed up the process is it possible to add the Quick Update menu item to the right-click of the Orphans grid row?

**Comment by BobRay** [comment]
Ok, I get it now. It's an interesting idea.

I'm not sure how QuickUpdate is launched. If I have time, I'll try to see if it's doable.

**Comment by BobRay** [comment]
I haven't forgotten this, but so far I've been unable to figure out how to add the feature.
